readme.md

```diff
@@ -78,4 +69,6 @@ $ ./cmd ibdsim -net=testnet # -net=testnet flag needed for testnet. Leave out fo
 [To resume, just do `./cmd ibdsim -net=testnet` again]
 ```
 
+* `ibdsim` is the CSN node and it will call genproofs and ask for blocks with the Utreexo accumulator proofs. It will receive the proofs and validate the inclusion.
```

Comment:
since this is the first use of this abbreviation in this file, it should be expanded

Reply:
Maybe it's a good idea to explain ibd somewhere (or maybe it could be assumed that people already know?) but here I'm referring to the ibdsim argument with the binary.

Reply:
yeah that too